shared/src/main/scala/VList.scala

```diff
@@ -56,7 +56,9 @@ class VList private (val lst: Seq[VAny])
       lst(Math.floorMod(ind, lst.length))
     else
       try lst(ind)
-      catch case _: IndexOutOfBoundsException => lst(ind % lst.length)
+      catch
+        case _: IndexOutOfBoundsException =>
+          if lst.isEmpty then 0 else lst(ind % lst.length)
```

Comment:
```scala
    if lst.isEmpty then 0
    else if ind < 0 then
      // floorMod because % gives negative results with negative dividends
      lst(Math.floorMod(ind, lst.length))
    else
      try lst(ind)
      catch
        case _: IndexOutOfBoundsException => lst(ind % lst.length)
```
I think the `if lst.isEmpty` check should go at the top in case the index is negative but the list is empty